ui/app/components/ui/tabs/tabs.stories.js

```diff
@@ -20,9 +20,7 @@ function renderTab (id) {
 
 export const twoTabs = () => {
   return (
-    <Tabs
-      defaultActiveTabIndex={number('Default Active Tab Index', 0, { min: 0 })}
-    >
```

Comment:
This didn't work before and it continues to not work: the default key isn't referenced after the element is mounted.